zebra-consensus/src/transaction/tests.rs

```diff
@@ -775,6 +777,47 @@ fn v4_with_sapling_outputs_and_no_spends() {
     });
 }
 
+/// Test if a V5 transaction with Sapling spends is accepted by the verifier.
+#[test]
+// TODO: Remove `should_panic` once V5 transaction verification is complete.
```

Comment:
do we have a ticket number or PR for this ?

Reply:
I couldn't find an issue to track full V5 transaction support, but I did add a `TODO` to the main code in [77337c1](https://github.com/ZcashFoundation/zebra/pull/2437/commits/77337c1736be2b6ec9e2b90878f527291f4363c5).

Reply:
> I couldn't find an issue to track full V5 transaction support

FYI that's #2379 - Validate remaining Orchard consensus rules in V5 transactions.